Mlem/Views/Shared/Components/Components/JumpButtonView.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  JumpButton.swift
+//  Mlem
+//
+//  Created by Sjmarf on 11/08/2023.
+//
+
+import SwiftUI
+import Dependencies
+
+struct JumpButtonView: View {
+    @State private var pressed: Bool = false
+    
+    @Dependency(\.hapticManager) var hapticManager
+
+    var onShortPress: () -> Void
+    var onLongPress: () -> Void
+    
+    var body: some View {
+        Button {} label: {
+            Image(systemName: "chevron.down")
+                .fontWeight(.semibold)
+                .foregroundStyle(.secondary)
+                .aspectRatio(contentMode: .fit)
+                .padding(16)
+                .background(
+                    ZStack {
+                        Circle()
+                            .fill(Material.thinMaterial)
+
+                        Circle()
+                            .strokeBorder(.tertiary.opacity(0.3), lineWidth: 1)
+                    }
+                )
+                .padding(10)
+                .scaleEffect(self.pressed ? 1.2 : 1.0)
+                .onTapGesture {
+                    hapticManager.play(haptic: .gentleInfo, priority: .high)
+                    onShortPress()
+                }
+                .onLongPressGesture(
+                    perform: {
+                        hapticManager.play(haptic: .gentleInfo, priority: .high)
+                        onLongPress()
+                    },
+                    onPressingChanged: { pressing in
+                        withAnimation(.spring) {
+                            self.pressed = pressing
+                        }
```

Comment:
`.spring` doesn't build on my machine--it needs to be called as `.spring()`

While we're here, how would we feel about `.interactiveSpring()` instead? The plain `.spring` feels a little mushy to me